spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -129,7 +128,88 @@ def write_recording(recording, save_path, acquisition_name):
             description='acquisition_description'
         )
         nwbfile.add_acquisition(ephys_ts)
+
         if os.path.exists(save_path):
             os.remove(save_path)
         with NWBHDF5IO(save_path, 'w') as io:
             io.write(nwbfile)
+
+
+
+
+class NwbSortingExtractor(se.SortingExtractor):
+    def __init__(self, path):
+        try:
+            from pynwb import NWBHDF5IO
+            from pynwb import NWBFile
+            from pynwb.ecephys import ElectricalSeries
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError("To use the Nwb extractors, install pynwb: \n\n"
+                                      "pip install pynwb\n\n")
+        #self._path = path
+        #with NWBHDF5IO(path, 'r') as io:
+        #    nwbfile = io.read()
+        #    NSX = se.NumpySortingExtractor()
+        se.RecordingExtractor.__init__()
+
+    @staticmethod
+    def write_sorting(sorting, save_path):
```

Comment:
Let's try including an optional dict argument for inserting optional metadata like `NWBFile.experimenter`.

Reply:
I opened an Issue for that